### DISCUSSION TITLE
Cannot save background (decorations) stuff when it targets body on v0.18.4

### DISCUSSION THREAD

**Discussion by handhikadj** [GrapesJS/grapesjs#4898] [General]
1. go to: https://codesandbox.io/s/great-kalam-v390lx
2. play with the background (or other decorations) on the body, like set the background color/etc
3. Refresh the page. you should see the color you set is not saved

is this a bug which related to https://github.com/GrapesJS/grapesjs/issues/1357 ?

**Comment by artf** [comment]
That bug was solved in next versions, so you have to upgrade
